### PR TITLE
Small docs update for WNet

### PR DESCRIPTION
**Pull request by C-Achard**
- [x] Change WNet to WNet3D
- [x] Make necessary changes to WNet guide
- [x] (Rename Review to Label)

### CODE REVIEW

File: docs/source/guides/detailed_walkthrough.rst [hunk 1]
```diff
@@ -116,7 +116,7 @@ In most cases this should left enabled.
    * **VNet** is a larger (than SegResNet) CNN from MONAI designed for medical image segmentation.
    * **TRAILMAP** is our implementation in PyTorch additionally trained on mouse cortical neural nuclei from mesoSPIM data.
    * **SwinUNetR** is a MONAI implementation of the SwinUNetR model. It is costly in compute and memory, but can achieve high performance.
-   * **WNet** is our reimplementation of an unsupervised model, which can be used to produce segmentation without labels.
+   * **WNet3D** is our reimplementation of an unsupervised model, which can be used to produce segmentation without labels.
```
Comment (MMathisLab) [inline docs/source/guides/detailed_walkthrough.rst:119]: It's our "extension"

File: docs/welcome.rst [hunk 2]
```diff
@@ -160,7 +160,7 @@ This plugin mainly uses the following libraries and software:
 
 * `pyclEsperanto`_ (for the Voronoi Otsu labeling) by Robert Haase
 
-* A custom re-implementation of the `WNet`_ by Xia and Kulis [3]_
+* A custom re-implementation of the `WNet3D`_ by Xia and Kulis [3]_
```
Comment (MMathisLab) [inline docs/welcome.rst:163]: More than a reimplementation, a new nn inspired by ...

**Review by MMathisLab — CHANGES_REQUESTED** [review]
Small comments

File: docs/welcome.rst (hunk 2 above)
Comment (C-Achard) [reply docs/welcome.rst]: Thanks

**Review by MMathisLab — APPROVED** [review]

File: docs/source/guides/detailed_walkthrough.rst [hunk 3]
```diff
@@ -116,7 +116,7 @@ In most cases this should left enabled.
    * **VNet** is a larger (than SegResNet) CNN from MONAI designed for medical image segmentation.
    * **TRAILMAP** is our implementation in PyTorch additionally trained on mouse cortical neural nuclei from mesoSPIM data.
    * **SwinUNetR** is a MONAI implementation of the SwinUNetR model. It is costly in compute and memory, but can achieve high performance.
-   * **WNet** is our reimplementation of an unsupervised model, which can be used to produce segmentation without labels.
+   * **WNet3D** is our extension of an unsupervised model, which can be used to produce segmentation without labels.
```
Comment (MMathisLab) [inline docs/source/guides/detailed_walkthrough.rst:119]: I guess best here to link to "unsupervised walkthrough?"

**Review by MMathisLab — CHANGES_REQUESTED** [review]
small new comments

**Review by MMathisLab — APPROVED** [review]

**Pull request merged**